cmd/snap/cmd_run.go

```diff
@@ -33,10 +34,12 @@ import (
 
 type cmdRun struct {
 	Positional struct {
-		SnapApp string `positional-arg-name:"<app name>" description:"the application to run, e.g. hello-world.env"`
+		SnapApp string `positional-arg-name:"<app name>" description:"the snap (e.g. hello-world) or application to run (e.g. hello-world.env)"`
 	} `positional-args:"yes" required:"yes"`
 
-	Command string `long:"command" description:"alternative command to run"`
+	Command  string `long:"command" description:"alternative command to run"`
+	Hook     string `long:"hook" description:"run a specific hook contained within the snap"`
```

Comment:
Yes, you can hide options with `hidden:"yes"`. Would you like `command` hidden as well?